superset/views/api.py

```diff
@@ -108,6 +112,31 @@ def time_range(self, **kwargs: Any) -> FlaskResponse:
             error_msg = {"message": _("Unexpected time range: %(error)s", error=error)}
             return self.json_response(error_msg, 400)
 
+    @api
+    @handle_api_exception
+    @has_access_api
+    @rison(ger_relative_time_range_schema)
+    @expose("/v1/relative_time_range/", methods=("GET",))
+    def relative_time_range(self, **kwargs: Any) -> FlaskResponse:
```

Comment:
Hey! Could we also accomplish the same adding a new optional field to the schema for `/v1/time_range/` and use that API instead?

Reply:
I was thinking about this and I'm not sure, because currently the rison schema in `/time_range` is just `string`. That worked fine when we only expected a time range string there, but not sure if we could make it work with 2 fields (time range and shift)